helpers/utils-helper.js

```diff
@@ -22,18 +22,31 @@ var Utils = {
     return promises;
   },
 
+  hasProperty: function(array, toFind) {
+    return Object.keys(array).some(function (key) {
+      return ~key.indexOf(toFind);
```

Comment:
- array is not an array but an object in your case, the bad naming can be source of error
  - hasProperty is badly named too, should be named "hasPropertyWithStartingChar"